package.json

```diff
@@ -21,43 +21,33 @@
     "Robin Berjon"
   ],
   "devDependencies": {
-    "@babel/core": "^7.9.0",
-    "@types/clipboard": "^2.0.1",
     "@types/pluralize": "0.0.29",
     "babel-eslint": "^10.1.0",
     "boxen": "^4.2.0",
     "chai": "^4.2.0",
     "chokidar": "^3.3.1",
     "clean-css": "^4.2.3",
-    "domReady": "github:requirejs/domReady#d85bdc38a6df868ead52ab3e80715aaf60e68765",
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.10.1",
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jasmine": "^4.1.0",
     "eslint-plugin-prettier": "^3.1.2",
-    "handlebars": "^4.7.3",
     "highlight.js": "^9.18.1",
     "hyperhtml": "^2.32.2",
     "idb": "^5.0.1",
-    "jasmine-core": "^3.5.0",
-    "jasmine-reporters": "^2.3.2",
+    "jasmine": "^3.5.0",
     "jquery": "^3.4.1",
     "karma": "^4.4.1",
     "karma-chrome-launcher": "^3.1.0",
-    "karma-detect-browsers": "^2.3.3",
     "karma-edge-launcher": "^0.4.2",
```

Comment:
this can probably go too I guess... it's just Chrome.

Reply:
@saschanaz might be using this?

Reply:
Nope 😁